### PR TITLE
Get session info

### PR DESCRIPTION
**Pull request by vhryhorchuk-amplify**
Added API call to retrieve info about the node by session-id.

TODO:
- [x] unit tests

### CODE REVIEW

**Review by jconstance-amplify — APPROVED** [review]

File: test/unit/proxy_test.py [hunk 1]
```diff
@@ -50,6 +50,18 @@ def test_create_session(self, mock_request):
...
 
+    @patch('amplium.api.proxy.GRID_HANDLER.unroll_session_id',
+           MagicMock(return_value=("test_session_id", "http://test_node_1:1234")))
+    @patch('amplium.api.proxy.send_request')
+    def test_get_session_info(self, mock_request):
+        """Test a correct request while getting Grid node info from session_id."""
+        proxy.get_session_info(session_id="ea88098b344441de443-4d7f48c3f749a")
+        mock_request.assert_called_once_with(
+            'GET',
+            "test_session_id",
```
Comment (apaschenko-amplify) [inline test/unit/proxy_test.py]: nitpick: consistent quotes

File: amplium/specs/swagger.yml [hunk 2]
```diff
@@ -68,6 +68,22 @@ paths:
           schema:
             $ref: '#/definitions/ok_response'
 
+  /proxy/api/session/{session_id}:
```
Comment (apaschenko-amplify) [inline amplium/specs/swagger.yml:71]: I'm confused now what url should I type in the browser's address bar to get the status. is it `mhcadminproxy-ci.aws.wgen.net/amplium/proxy/api/session/{session_id}`?

Comment (apaschenko-amplify) [reply amplium/specs/swagger.yml:71]: or it's not intended for humans even?

**Review by apaschenko-amplify — COMMENTED** [review]
Code looks good to me, though I'd use some explanation on why the feature is needed and who's the user

File: amplium/specs/swagger.yml (hunk 2 above)
Comment (vhryhorchuk-amplify) [reply amplium/specs/swagger.yml:71]: We still have `/status` endpoint, which gives us info about running Selenium Grid.
The new endpoint can get us info about a specific Selenium node, which is associated with Selenium session_id.
Let's say, I create a new session by starting tests, and I would like to know where my tests are running by that session id, so, if something breaks, I can debug/ ssh to the node.

File: test/unit/proxy_test.py (hunk 1 above)
Comment (vhryhorchuk-amplify) [reply test/unit/proxy_test.py]: \+

**Review by jconstance-amplify — APPROVED** [review]

**Pull request merged**